petpals/forum/post/templates/post.html

```diff
@@ -56,46 +65,39 @@ <h5 class="mt-1">{{ post.title }}</h5>
-                      data-reply-id={{ reply.reply_id }}
-                    >
-                      <i
-                        class="{{ 'fa-solid' if current_user.is_authenticated and reply.likes.filter_by(user_id=current_user.id).first().liked else 'fa-regular' }} fa-heart me-1"
-                        style="color: crimson;"
-                        aria-hidden="true"
-                      ></i>
-                      <span>{{ reply.likes.filter_by(liked=True).all() | length }}</span>
-                    </button>
-                    {% if current_user.is_authenticated and current_user.id == reply.user_id %}
-                      <a
-                        href="{{ url_for('forum_router.post_router.reply_router.reply_edit', post_id=post.post_id, reply_id=reply.reply_id) }}"
-                        class="btn btn-sm text-muted"
-                      >
-                        Edit Reply
-                      </a>
-                    {% endif %}
+        <!--Reply Cards Container-->
+        <div id="reply{{ reply.reply_id }}" class="container mt-0 bg-info border border-6 border-info p-0">
+          <!--Reply Content-->
+          <div class="forum-card mt-2 mb-2 p-1 border border-3 border-light">
+            <div class="forum-card-body">
+              <div class="media forum-item">
+                <a href="{{ url_for('profile_router.profile_user', username=reply.author.username) }}"
+                  class="card-link">
+                  <img src="{{ reply.author.image_path }}" class="rounded-circle" width="50 " alt="User " />
+                </a>
+                <div class="media-body ml-3">
+                  <a href="{{ url_for('profile_router.profile_user', username=reply.author.username) }}"
+                    class="text-secondary">{{ reply.author.username }}</a>
+                  <small class="text-muted ml-2">{{ reply.timestamp }}</small>
+                  <div class="mt-3 font-size-sm mb-4">
+                    {{ content_markup(reply.content) }}
                   </div>
+                  <button class="like btn btn-xs text-muted has-icon me-3" data-reply-id={{ reply.reply_id }}>
+                    <i class="{{ 'fa-solid' if current_user.is_authenticated and reply.likes.filter_by(user_id=current_user.id).first().liked else 'fa-regular' }} fa-heart me-1"
+                      style="color: crimson;" aria-hidden="true"></i>
+                    <span>{{ reply.likes.filter_by(liked=True).all() | length }}</span>
+                  </button>
+                  {% if current_user.is_authenticated and current_user.id == reply.user_id %}
+                  <a href="{{ url_for('forum_router.post_router.reply_router.reply_edit', post_id=post.post_id, reply_id=reply.reply_id) }}"
+                    class="btn btn-sm text-muted">
+                    Edit Reply
+                  </a>
+                  {% endif %}
                 </div>
               </div>
             </div>
           </div>
+        </div>
```

Comment:
From what I can see, none of these diffs are actually changes.
Please change them back and avoid making minor formatting changes in the future. It makes it hard to review :(
If you want to reformat something, save that for a separate PR :)
(It looks like a mistake though because it's not formatted correctly anyway)